src/app/pages/ConnectorDetailsPage/ConfigurationTab/ErrorHandlerStep.tsx

```diff
@@ -1,26 +1,47 @@
 import { StepBodyLayout } from '@app/components/StepBodyLayout/StepBodyLayout';
-import { createValidator } from '@utils/createValidator';
-import _ from 'lodash';
-import React, { useEffect, useState } from 'react';
-import { FC } from 'react';
+import { toHtmlSafeId } from '@utils/shared';
+import React, { FC, useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
-import { JSONSchemaBridge } from 'uniforms-bridge-json-schema';
 
 import {
   Form,
   FormGroup,
   Popover,
   Select,
   SelectOption,
+  SelectOptionObject,
   SelectVariant,
   Text,
   TextInput,
   TextVariants,
 } from '@patternfly/react-core';
 import { HelpIcon } from '@patternfly/react-icons';
 
-export type ErrorHandler = {
-  [key: string]: any;
+import { Connector } from '@rhoas/connector-management-sdk';
+
+// error_handler does not yet seem to be available in the SDK
+export type LogErrorHandler = {
+  log: object;
+};
+
+export type StopErrorHandler = {
+  stop: object;
+};
+
+export type DLQErrorHandlerTopic = {
+  topic: string;
+};
+
+export type DLQErrorHandler = {
+  dead_letter_queue: DLQErrorHandlerTopic;
+};
+
+export type ErrorHandler = LogErrorHandler | StopErrorHandler | DLQErrorHandler;
```

Comment:
Thanks for adding this :+1:

Reply:
@indraraj yeah, I guess we may need to introduce some interim model where we can extend the actual API model as needed to deal with these situations where the API is in development.  Probably a needed improvement is to pull these types out of here and create a `models` directory where we can keep these extended models.

Unless there's some better way to accomplish this, could be worth some more research.